__tests__/integration/html.ts

```diff
@@ -79,9 +79,7 @@ describe("sitemap integration", () => {
           }
 
           const alt = tag.attr("alt");
-          if (!alt) {
-            altTagMissing.push(name);
-          }
+          if (!alt && !name.match(/^data:image/)) altTagMissing.push(name);
```

Comment:
Why shouldn't `data:image` have an `alt` name for SEO?

Reply:
It's a Next thing I don't quite understand - I think they are making a placeholder image (transparent gif) that displays until our real image loads?  I could cross-check this with `aria-hidden="true"` perhaps

<img width="1575" alt="Screen Shot 2020-11-23 at 10 38 19 AM" src="https://user-images.githubusercontent.com/303226/100001434-03eb2680-2d78-11eb-8ecd-fdb639202d72.png">

Reply:
yeah. or maybe that role=presentation
never seen that before.

Reply:
Yeah! That works (see https://github.com/grouparoo/www.grouparoo.com/pull/92/commits/99cbd22ee30ce8bcc68931ec8decdf2558e41f7d)